massa-db-worker/src/massa_db.rs

```diff
@@ -141,7 +137,7 @@ where
                                 }
                                 updates
                             }
-                            None => {
+                            _ => {
```

Comment:
I would add more context and explain how / when it can happen so a reader would pay attention to this point :)

Reply:
Added some comments, is it better? :)